Entitas.Unity/Assets/Entitas/Unity/Editor/EntitasEditorLayout.cs

```diff
@@ -55,4 +58,4 @@ public static class EntitasEditorLayout {
             EditorGUILayout.EndHorizontal();
         }
     }
-}
```

Comment:
The Github editor snuck in this change. Generally you want a newline at the end of a file.